open-company/open-company-web
# Finances editing

cards:
- https://trello.com/c/L6LB7xO3
- https://trello.com/c/SEaCUpna

To test:
- import bago company from api
- open /companies/bago
- do you see the charts for finances and growth?
- can you navigate back in the chart on each metric/link?
- does all the values seem fine?
- try to edit the finances data
- open buffer and try to edit the finances data: here you should see the last 12 month editable even if there are less than 12 values initially
- change some values and some other using the more button at the bottom of the table
- save
- is everything ok?
- run the tests or check the CI
- merge
